### PR TITLE
Booleans

### PR DESCRIPTION
**Pull request by TechnichalDoge**
Pretty small, but this adds in constants for true and false, to make code look prettier by default.

### CODE REVIEW

File: lua/entities/gmod_wire_expression2/core/custom/cfc_array.lua [hunk 1]
```diff
@@ -2,14 +2,14 @@ E2Lib.RegisterExtension( "cfc_e2_lib", true )
 
 e2function array array:shuffled()
     local size = #this
-    for i = size, 1, -1 do
```
Comment (PLally) [inline lua/entities/gmod_wire_expression2/core/custom/cfc_array.lua]: None of these comments are needed

**Comment by TechnichalDoge** [comment]
> you really only need the 1 file that registers the constants in this pr

i didnt mean to add this, i didnt even touch the arrays, its just my mistake on how  i did through it

File: lua/entities/gmod_wire_expression2/core/custom/cfc_array.lua (hunk 1 above)
Comment (TechnichalDoge) [reply lua/entities/gmod_wire_expression2/core/custom/cfc_array.lua]: done

**Comment by TechnichalDoge** [comment]
By the by, the changes that are requested are done

**Pull request closed without merging**